manifest.json

```diff
@@ -1,47 +1,43 @@
 {
     "description": "Tries to play links in MPV",
-    "manifest_version": 2,
+    "manifest_version": 3,
     "name": "ff2mpv",
-    "version": "5.1.1",
-    "options_ui": {
-        "page": "options/options.html",
-        "open_in_tab": false
-    },
+    "version": "6.0.0",
     "icons": {
         "16": "icons/icon_16x16.png",
         "32": "icons/icon_32x32.png",
         "48": "icons/icon_48x48.png",
         "64": "icons/icon_64x64.png",
         "256": "icons/icon_256x256.png"
     },
-    "browser_action": {
+    "action": {
         "default_icon": "icons/icon_48x48.png",
         "default_title": "Play the current URL in MPV"
     },
     "commands": {
-        "_execute_browser_action": {
-            "suggested_key": {
-                "default": "Ctrl+Shift+V"
-            },
+        "_execute_action": {
             "description": "Play the current URL in mpv"
         }
     },
-    "applications": {
+    "browser_specific_settings": {
         "gecko": {
             "id": "ff2mpv@yossarian.net",
             "strict_min_version": "50.0"
```

Comment:
I imagine this will now be higher, right? Or could possibly be removed entirely, since only newer FF will support MV3 anyways.